connectivity/FEATURE_BLE/libraries/TARGET_CORDIO_LL/CMakeLists.txt

```
@@ -1,7 +1,7 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-target_include_directories(mbed-os
+target_include_directories(mbed-os-ble-cordio
```

```suggestion
target_include_directories(mbed-os-ble-cordio_ll
```
I think you need to create another CMake target library here and call it `mbed-os-ble-cordio_ll`. All source and include dir (and its sub-dirs) should be added to this new target.